File: Abc.java
public class Abc {
 public static void main(String[] args) {
  String abc = "";

  for (int i = 0; i < 3; i++) {
   abc += (char) ('a' + i);
  }
  int j = 3;
  while (j != 0) {
   abc += (char) ('d' - j--);

  }

  char lettre = 'a';
  do {
   abc += lettre;
   lettre++;
  } while (lettre != 'd');
  System.out.println(abc);

 }

}

File: abc.cpp
#include <iostream>
using namespace std;
int i = 0;

int main(int argc, char const *argv[])
{
 /* code */
 for (i = 0; i < 3; i++)
 {
  cout << (char)('a' + i);
 }

 int j = 3;
 while (j != 0)
 {
  cout << (char)('p' - j--);
 }

 char lettre = 'a';
 do
 {
  cout << lettre;
  lettre++;
 } while (lettre != 'd');

 cout << endl;

 return 0;
}
